drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
```

Comment:
I'd prefer the actor being swapped here.
c should set its unset fields from the parameter config, not the other way around.

Also, when this function is being called in the loop, the config object has already been constructed. We should not look into options again.